docs/specs/devcontainerjson-reference.md

```diff
@@ -2,32 +2,32 @@
 
 The `devcontainer.json` file contains any needed metadata and settings required to configurate a **development container** for a given well-defined tool and runtime stack. It can be used by [tools and services that support the dev container spec](supporting-tools.md) to create a **development environment** that contains one or more **development containers**.
 
-Metadata properties marked with a 🏷️ can be stored in the `devcontainer.metadata` **container image label** in addition to `devcontainer.json`. This label can contain an array of json snippets that will be automatically merged with `devcontainer.json` contents (if any) when a container is created.
+Metadata properties marked with a 🏷 can be stored in the `devcontainer.metadata` **container image label** in addition to `devcontainer.json`. This label can contain an array of json snippets that will be automatically merged with `devcontainer.json` contents (if any) when a container is created.
```

Comment:
ack! I updated the emoji since it didn't look right on windows and looks like the PR reset it.   🏷 is the new one.  Also, find and replace works with emojis in VS Code so it should be easy to fix. :smile:

Reply:
Oh no I don't know how I reverted that! Some mistake in a merge conflict I think 😄 

Thanks for catching, fixed!